src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comment } from '../../types/CommentInfo';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[]
+};
+
+const commentListSending = (comments: Comment[]) => {
```

Comment:
specify what return this function.
also not necessary to make this function, 
you can move this login in CommentList component